### PR TITLE
chore: jest update

### PR DESCRIPTION
**Pull request by clintonmedbery**
We have to update the snapshots as Jest updated the way it does snapshots.
https://jestjs.io/blog/2022/08/25/jest-29

# Description

This updates jest. Pretty straight forward.

## Which issue(s) does this PR relate to?

 - https://jira.d2iq.com/browse/D2IQ-

## Checklist

- [x] If any new components were added, there are exported from `packages/index.ts`
- [x] This PR is associated with a JIRA and mentions in the commit message footer ("Closes …")
- [ ] This PR contains breaking changes and states in the commit message body ("BREAKING CHANGE: …")
- [x] I have reviewed the changes and provided detail to the sections above

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,7 +88,7 @@
     "@storybook/theming": "6.5.10",
     "@svgr/webpack": "5.5.0",
     "@types/enzyme": "3.10.12",
-    "@types/jest": "27.4.0",
+    "@types/jest": "28.1.8",
```
Comment (clintonmedbery) [inline package.json:91]: Apparently emotion/jest is not ready for types 29.x yet.

Comment (vacas5) [inline package.json:91]: both @jest/types and @types/jest?

Comment (clintonmedbery) [reply package.json:91]: Whoops... 🙈 fixed

Comment (vacas5) [reply package.json:90]: is there a @types/jest for 29?

**Review by vacas5 — APPROVED** [review]

**Pull request merged**